onnxruntime/test/providers/cpu/nn/qlinearconv_op_test.cc

```diff
@@ -377,12 +394,18 @@ class QLinearConvOpTester {
     const int64_t* output_shape = Y_shape.data() + 2;
     Y_data.resize(ShapeSize(Y_shape));
 
+    auto calc_size = [](size_t N, const int64_t* shape) -> int64_t {
```

Comment:
The code code look more like the existing style if you added:

  static size_t ShapeSize(const int64_t* shape, size_t rank) {
    return static_cast<size_t>(std::accumulate(shape, shape + rank, 1LL, std::multiplies<int64_t>()));
  }

Reply:
Yeah. Let use std::accumulate directly.